front/src/modules/timesStops/hooks/useOutputTableData.ts

```diff
@@ -89,11 +90,15 @@ function useOutputTableData(
             selectedTrainSchedule,
             pathStepId
           );
+          const calculatedArrival = secToHoursString(opPoint.time, true);
```

Comment:
Why converting to a string, and then comparing number? Wouldn't it be more logical to compare first numbers directly?

Reply:
I’m now comparing the numbers directly before converting the TimeString.
added some tests as well